src/edu/csus/ecs/pc2/ui/SelectJudgementPaneNew.java

```diff
@@ -1171,7 +1174,7 @@ public void actionPerformed(java.awt.event.ActionEvent e) {
     }
 
     private String getExecuteDirectoryName() {
-        Executable tempEexecutable = new Executable(getContest(), getController(), run, runFiles);
+        Executable tempEexecutable = new Executable(getContest(), getController(), run, runFiles, null);
```

Comment:
No change required, note for future development
OOP note, instead of requiring a null param (esp at end) create a new constructor

Reply:
Noted.  I just want to be sure anyone that uses this class knows how it works and what a null frame implies; requiring that extra frame param enforces that contract (to some extent).  That is all.  I think we spoke about this offline - in C++ I might make that last argument a default argument, eg( ...  , frame = NULL).